dirty.rb

```diff
@@ -0,0 +1,38 @@
+a =     'b'
+eval '1+1'
+"#{1}"
+begin
+  foo
+rescue
+end
+begin
+begin
+begin
+begin
+  fooaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+rescue
```

Comment:
__[development][rubocop]__
* Do not suppress exceptions. [:ref](https://github.com/bbatsov/ruby-style-guide#dont-hide-exceptions)